Gemfile

```diff
@@ -110,6 +110,7 @@ group :development do
   gem "bullet", "~> 5.9"
   gem "bundler-audit", "~> 0.6"
   gem "derailed_benchmarks", "~> 1.3"
+  gem "foreman", github: "andrewmcodes/foreman", ref: "b64e401", require: false # Switch to stable release when Thor dependecy is updated to 0.20+
```

Comment:
Will this ref still work if the branch which it belongs to is removed?

Reply:
If the branch is deleted I think this would start failing. If you fork `andrew/foreman` under `thepracticledev` I'll update the repo until https://github.com/ddollar/foreman/pull/727 is merged.

Reply:
Here you go
```suggestion
  gem "foreman", github: "thepracticaldev/foreman", ref: "b64e401", require: false # Switch to stable release when Thor dependecy is updated to 0.20+
```

Reply:
@maestromac The fork doesn't have the `update-thor` branch so you'll need to copy that too.

Reply:
That won't be necessary since the ref match. I've tested it locally.

Reply:
Cool!

Reply:
Suggested changes probably shouldn't be used for `Gemfile` changes as the `Gemfile.lock` will be outdated. I've updated this branch's lock file but I missed it initially.